### DISCUSSION TITLE
Building from source

### DISCUSSION THREAD

**Discussion by timheuer** [AdguardTeam/AdGuardHome#7083] [General]
I'm trying to build from source using a devcontainer (have the go dependencies as using the go devcontainer).

When running `make`:
```bash
env CHANNEL='development' COMMIT='$( git rev-parse --short HEAD )' DIST_DIR='dist' GO="${GO:-go}" GOAMD64="v1" GOPROXY='https://goproxy.cn|https://proxy.golang.org|direct' GOSUMDB='sum.golang.google.cn' GOTOOLCHAIN='go1.22.4' GPG_KEY='devteam@adguard.com' GPG_KEY_PASSPHRASE='not-a-real-password' PATH="${PWD}/bin:$( "${GO:-go}" env GOPATH )/bin:${PATH}" RACE='0' SIGN='1' NEXTAPI='0' VERBOSE="${VERBOSE:-0}" VERSION='v0.0.0'  "/bin/sh" ./scripts/make/go-deps.sh
: not foundake/go-deps.sh: 2: 
: not foundake/go-deps.sh: 7: 
: bad variable nameeps.sh: 9: readonly: verbose
make: *** [Makefile:106: go-deps] Error 2```

**Comment by ainar-g** [comment]
Are you just running `make` with no targets and macros?  Have you read the [build scripts documentation](https://github.com/AdguardTeam/AdGuardHome/tree/master/scripts#make-makefile-scripts)?

**Comment by timheuer** [comment]
Yes. Following the instructions from the readme
https://github.com/AdguardTeam/AdGuardHome/tree/master?tab=readme-ov-file#building

Is that wrong and should have a better sample?
